src/common/components/Icons/ShieldedBalance.tsx

```diff
@@ -0,0 +1,296 @@
+import React from 'react'
+
+export type TShieldedBalanceProps = {
+  size?: number
+  className?: string
+  variant?: 'active' | 'inactive'
+}
+
+export const ShieldedBalance = ({
+  size = 70,
+  className = 'text-gray-88',
+  variant = 'active',
+}: TShieldedBalanceProps): JSX.Element => {
+  return (
+    <svg
+      width={size}
+      viewBox='0 0 70 70'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+      className={className}
+    >
+      <path
+        d='M5 6H64V61.0667H5V6Z'
+        stroke={variant === 'active' ? 'url(#paint0_linear)' : '#B9BFCB'}
```

Comment:
It's bad to hardcode colors in common icon component, but it can not be solved with tailwind simply, and design most likely has this icon only with specific colors, so let it be so for now

Reply:
okay